site/rebase-image.md

```diff
@@ -47,11 +47,20 @@ Each section can contain one or more valid Dockerfile commands and would look li
 ```dockerfile
 [after-fmw-install]
 RUN rm /some/dir/unnecessary-file
+COPY --chown=oracle:oracle files/my_additional_file.txt /u01
 
 [final-build-commands]
 LABEL owner="middleware team"
 ```
 
+#### `--additionalBuildFiles`
```

Comment:
We should specify that the copy command has to be in the context of valid sections: before-jdk-install, after-jdk-install, before-fmw-install, after-fmw-install, before-wdt-command, after-wdt-command, final-build-commands. Example:
[final-build-commands]
COPY --chown=oracle:oracle files/convenience.sh /my/internal/image/location